src/components/AppBar.tsx

```diff
@@ -61,14 +74,47 @@ export const AppBar = () => {
       style={{ zIndex: 1 }}
     >
       <div className="ml50 flex">
-        <Link to={routesEnum.landingPage}>
+        <Link to={routesEnum.landingPage} className="mx10">
           <EthLogo src={EthDiamond} alt="eth-diamond" />
-          <Heading level={4} margin="none" style={{ padding: '8px 0' }}>
+          <Heading level={4} margin="none" style={{ padding: '12px 0' }}>
             Eth2 Launch Pad{' '}
             {IS_MAINNET ? `` : ` for ${ETH2_NETWORK_NAME} testnet`}
           </Heading>
         </Link>
+        <Link to={routesEnum.acknowledgementPage} className="mx10">
```

Comment:
Since some items are two-word or three-word, what do you think about adding some hover effect to the navbar items so that it's more clear to see which link I'm pointing to?